patchwork-fml/src/main/java/net/minecraftforge/versions/mcp/MCPVersion.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.versions.mcp;
+
+import net.minecraft.SharedConstants;
+
+public class MCPVersion {
+	public static String getMCVersion() {
+		return SharedConstants.getGameVersion().getName();
```

Comment:
The indentation looks different for this class than all the others

Reply:
looks the same in ide and they are all tabs, for some reason github is displaying them wrong :hmm: